deploy/003-price-oracle-configure-feeds.ts

```diff
@@ -9,6 +9,6 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
 };
 
 func.tags = ["Oracle"];
-func.skip = async (hre: HardhatRuntimeEnvironment) => hre.network.name !== "hardhat";
+func.skip = async (hre: HardhatRuntimeEnvironment) => hre.network.live;
```

Comment:
We need to remove line 7

Reply:
We don't actually need to define and extend `DeployFunction` as it is defined and the logic is in the deployment we import from... pushed the fix for that